cryptoki/src/context/mod.rs

```diff
@@ -158,7 +157,7 @@ impl Pkcs11 {
     }
 
     /// Open a new session with no callback set
-    pub fn open_session_no_callback(&self, slot_id: Slot, flags: SessionFlags) -> Result<Session> {
-        session_management::open_session_no_callback(self, slot_id, flags)
+    pub fn open_session_no_callback(&self, slot_id: Slot, read_write: bool) -> Result<Session> {
```

Comment:
:thinking: I'm not really too much into having boolean parameters since it's not obvious what `open_session_no_callback(..., true)` does (an enum with `READ_WRITE` or `READ_ONLY` would be more... readable) but on the other hand it's commonly used and the IDE can offer hints.

Or even export two functions: `open_readonly_session_no_callback` or `open_readwrite_session_no_callback`...

Btw very nice to see you providing explicit reasoning in the commit message for SessionFlags refactor: https://github.com/parallaxsecond/rust-cryptoki/pull/68/commits/67e758928631c68f3ed4db601df8e58dd95ad50a :clap: thanks!